.nais/naiserator-prod.yaml

```diff
@@ -55,6 +55,7 @@ spec:
         - host: "syfo-tilgangskontroll.prod-fss-pub.nais.io"
       rules:
         - application: syfobehandlendeenhet
+        - application: syfooversiktsrv-redis
```

Comment:
Ser ut som redis-cachen ikke har vært i bruk (mye "Got connection error when storing in redis" i loggen).